nuxeo-runtime/nuxeo-connect-standalone/src/test/java/org/nuxeo/launcher/connect/TestConnectBroker.java

```diff
@@ -76,6 +79,10 @@
 
     final File nuxeoHome = new File("target/launcher");
 
+    public static final PackageState NON_EXISTENT = null;
```

Comment:
confusing shortcut versus `PackageState.UNKNOWN` while here it means: `no package found, so no associated package state in the list retrieved by the internal code`...
bof